src/index.js

```diff
@@ -90,29 +90,37 @@ function buildArgs(options) {
 // function creates the conf object to be written to that temp file before
 // passing to jsdoc as a command line argument.
 function buildConf(options) {
-    let opts = options || {};
-
+    const opts = options || {};
+    const config = typeof opts.config === 'object' && !Array.isArray(opts.config)
+        ? opts.config
+        : {};
     // updating default JSDoc configuration.
     // see http://usejsdoc.org/about-configuring-jsdoc.html
-    return {
+    return _.defaultsDeep(config, {
         tags: {
             allowUnknownTags: typeof opts.allowUnknownTags === 'boolean'
                 ? opts.allowUnknownTags
                 : true,
-            dictionaries: !Array.isArray(opts.dictionaries)
-                ? ['jsdoc', 'closure']
-                : opts.dictionaries
+            dictionaries: Array.isArray(opts.dictionaries)
```

Comment:
Swapping the ternary conditional here in order to make all conditionals verify truthiness instead of falseness to make the code consistent.